views/partials/header.ejs

```diff
@@ -237,84 +270,92 @@
+										<%=userNotifications[i].text%>
+									</p>
+								</a>
 
-								<% if(locals.userNotifications) { for(var i = userNotifications.length - 1; i >= 0; i--){ %>
-									<% if(userNotifications[i].seen === false || userNotifications[i].seen=== undefined){ %>
-										<li class="notificationItem unseen" idOfNotif=<%=userNotifications[i]._id%>>
-									<% } else{ %>
-										<li class="notificationItem" idOfNotif=<%=userNotifications[i]._id%>>
-									<% } %>
-											<a  href="<%=userNotifications[i].link%>"><p class="notificationText" style="white-space: normal;"><%=userNotifications[i].text%></p></a>
+								<%
+									<!-- // var dateToShow = new Date(userNotifications[i].date).toDateString();
+									var dateToShow = new Date(userNotifications[i].date).toLocaleString('en-GB',{hour12: true});
 
-											<%
-												// var dateToShow = new Date(userNotifications[i].date).toDateString();
-												var dateToShow = new Date(userNotifications[i].date).toLocaleString('en-GB',{hour12: true});
+									// if(new Date().toDateString() === dateToShow){
+									// 	dateToShow = new Date(userNotifications[i].date).
+									// } -->
+								%>
 
-												// if(new Date().toDateString() === dateToShow){
-												// 	dateToShow = new Date(userNotifications[i].date).
-												// }
-											%>
 
+								<p><span class="notificationDate">
+										<%=dateToShow%></span>
+									<span class="notificationHide">Hide<span class="hidden userNotificationIdToRemove">
+											<%=userNotifications[i]._id%></span></span>
+								</p>
+								<!--<p><%= userNotifications[i].link %></p>-->
 
-											<p><span class="notificationDate"><%=dateToShow%></span>
-											<span class="notificationHide" >Hide<span class="hidden userNotificationIdToRemove" ><%=userNotifications[i]._id%></span></span>
-											</p>
-											<!--<p><%= userNotifications[i].link %></p>-->
+							</li>
 
-										</li>
+							<% if(!(i === 0)) { %>
+							<hr>
+							<% } %>
 
-										<% if(!(i === 0)) { %>
-											<hr>
-										<% } %>
+							<% } %>
+							<% } %>
+						</ul>
 
-									<% } %>
-								<% } %>
-							</ul>
 
-
-						</li>
+					</li>
 					<% } %>
 
 
 					<% if(locals.optionsCog){ %>
-						<li>
-							<a href="#" role="button" data-toggle="modal" data-target="#optionsModal" class="optionsCog">
-								<span class="glyphicon glyphicon-cog"></span>
-							</a>
-						</li>
+					<li>
+						<a href="#" role="button" data-toggle="modal" data-target="#optionsModal" class="optionsCog">
+							<span class="glyphicon glyphicon-cog"></span>
+						</a>
+					</li>
 					<% } %>
 					<!-- <li><a href="#">Hi there, <%=currentUser.username%></a></li> -->
-					<li><a href="/profile/<%=currentUser.username%>" class="" id="originalUsername"><%=currentUser.username%></a></li>
+					<li><a href="/profile/<%=currentUser.username%>" class="" id="originalUsername">
```

Comment:
Here is the issue. I have no idea why having this on two lines instead of one causes any problems. But somehow it does... The issue wasn't present on Chrome, only Firefox. When I make this one line again, it works fine...

Reply:
Just pushed up the fix. Thank you for your help! (In the future please only include line changes relevant to your commit!)